Procfile

```diff
@@ -1,3 +1,7 @@
-web: python manage.py serve -d -r -h 0.0.0.0 -p 6666
-worker: celery -A udata.worker worker
-beat: celery -A udata.worker beat
+# Uncomment lines given your setup and workflow:
+# mongodb: mongod --config /usr/local/etc/mongod.conf
+# es: elasticsearch --config=/usr/local/opt/elasticsearch17/config/elasticsearch.yml
```

Comment:
It's not very cross-plateform. I'll but them (es and mongodb) lower in the file with a comment line saying: for Homebrew